src/lib/db.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const config = require("./config");
+
+const connect = () => {
+  return new Promise(async (resolve, reject) => {
+    const { user, password, host } = config.db;
+

```

Comment:
add the following line before the mongose.connect mongoose.set("strictQuery", false);